source/account_structure/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Account structure 
+weight: 55
+---
+
+# Account structure 
+
+If you have multiple services integrated, you can treat them separately or as
+a combined single service. You can use the GOV.UK Pay platform to set this this within
+GOV.UK Pay and within your Payment Service Provider (PSP). 
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/account_structure.png)
+
+You may have 2 services, for example for parking permits and parking fines.
+The diagram takes this example and illustrates the different possible
+structures you can have:
+
+### Type 1 (GOV.UK Pay separate, PSP separate)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+both GOV.UK Pay and the PSP.
+
+### Type 2 (GOV.UK Pay combined, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as a single combined
+service in GOV.UK Pay and in the PSP.
+
+### Type 3 (GOV.UK Pay separate, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+GOV.UK Pay and as a single combined service in the PSP. 
+
+## Editing GOV.UK Pay services 
+
+Each service defined in GOV.UK Pay has its own API.
+
+In GOV.UK Pay, within each service you can edit: 
+
+* user permissions 
+* email template to user
+* accepted card types
+* name that appears on the payment pages 
+* accept 3DS 
+
+## Editing PSP services 
+
+Within each service for the PSP, you can edit: 
+
+* what appears on bank statement of your users 
```

Comment:
i'm not totally sure this is true - @tillwirth can you confirm?

Reply:
oh i meant about customising what's on the bank statement! sorry i wasn't clear.

Reply:
Just a note for completeness: this should be OK as it stands